stockflux-securities-master/src/components/TextField.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./TextField.css";
+
+const TextField = props => {
+  return (
+    <input
+      {...props}
+      className={
+        "input-form-input" +
+        (props.readOnly ? " read-only" : "") +
+        ` ${props.className}`
```

Comment:
Since you're using backticks here which allows for string interpolation, you can simplify the whole `className`:

    className={`input-form-input ${readOnly && "read-only"} ${className}`}

Reply:
I think that will result in an unwanted space at the end if the props are not defined. I'll do a version of that but fix that issue

Reply:
it looks like it doesn't like me using the `condition && value`  method in the className.  I've asked Matt if he knows why but he said he didn't know. 

` readOnly ? " read-only"  : ""`
does work though.

Reply:
> I think that will result in an unwanted space at the end if the props are not defined. I'll do a version of that but fix that issue

It will, yes, but you have that anyway in your current version with ` ${props.className}`. If `className` is not explicitly defined when using your component, a trailing space will be left too. I'd be interested to have a look at why the `condition && value` approach isn't working